admin/class-smaily-for-wp-admin.php

```diff
@@ -120,7 +120,7 @@ public function smaily_subscription_block_init( $screen ) {
 			true
 		);
 
-		wp_add_inline_script( $this->plugin_name, 'var autoresponders = ' . json_encode( $autoresponders ) . ';' );
+		wp_add_inline_script($this->plugin_name , "var autoresponders = '" . wp_json_encode( $autoresponders ) . "';", 'before' );
```

Comment:
What happens if an autoresponder name contains a single-quote?

Reply:
Not ideal, but I escape all quotes. https://github.com/sendsmaily/sendsmaily-wordpress-plugin/pull/142/commits/dddface88e10d19fc9fb837bbaaefbc00976473f Its a quick fix, but gets rid of the current problem.

We should probably rewrite the block sections so that we also have a source code in the repo. This allows us to add new features and fix problems directly in that code and then rebuild the minified version.